safe-global/safe-react
# feat: filter historical transactions

## What it solves
Requesting filtered transactions.

## How this PR fixes it
The chosen filter is now applied to the historical transactions, as well as being appended to the URL (with the added benefit of being able to share).

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/172583485-049ae7c2-445b-4c5d-a814-11e6c8841f96.png)

## How to test it
Filter the transactions and observe the filtered data. The filter should also be appended to the URL. Opening the filter-appended URL should load the historical transactions, filtered.
